### PR TITLE
OpenEMR BUGFIX: #462 Address book permits saving an empty form

### PR DESCRIPTION
**Pull request by visolve-openemr**

### CODE REVIEW

File: interface/usergroup/addrbook_edit.php [hunk 1]
```diff
@@ -107,157 +107,174 @@ function typeSelect(a) {
...
-    "city2 = "        . invalue('form_city2')        . ", " .
-    "state2 = "       . invalue('form_state2')       . ", " .
-    "zip2 = "         . invalue('form_zip2')         . ", " .
-    "phone = "        . invalue('form_phone')        . ", " .
-    "phonew1 = "      . invalue('form_phonew1')      . ", " .
-    "phonew2 = "      . invalue('form_phonew2')      . ", " .
-    "phonecell = "    . invalue('form_phonecell')    . ", " .
-    "fax = "          . invalue('form_fax')          . ", " .
-    "notes = "        . invalue('form_notes')        . " "  .
-    "WHERE id = '" . add_escape_custom($userid) . "'";
```
Comment (bradymiller) [inline interface/usergroup/addrbook_edit.php]: Will this lose other items on the form, though. For example, if a address is entered in? Or will they stay populated?

Comment (bradymiller) [inline interface/usergroup/addrbook_edit.php]: If this is the case(ie. lost population of filled out items), would it be better to test for these items prior to the submit?

**Comment by mdsupport** [comment]
Hard to pin down changes but fix seems to address POSTed data.  Wouldn't it be better to also disable submit button/event using client-side validation?

**Comment by robertdown** [comment]
Closing this since outdated PR

**Pull request closed without merging**